relishable.gemspec

```diff
@@ -13,7 +13,9 @@ Gem::Specification.new do |s|
   s.files = Dir["lib/**/*.rb"] + Dir["Gemfile*"]
   s.require_paths = ["lib"]
   s.add_dependency "fog",           "~> 1.23.0"
```

Comment:
You can just remove the fog dependency altogether, probably best to do so actually. `fog-aws` should include everything you need.